samples/unit-tests/series-venn/members/demo.js

```diff
@@ -667,4 +667,40 @@ QUnit.test('sortByTotalOverlap', function (assert) {
         0,
         'should return 0 when a is equal to b.'
     );
+});
+
+QUnit.module('nelder-mead', () => {
+    const vennUtils = Highcharts.seriesTypes.venn.prototype.utils;
+    const NelderMeadModule = vennUtils.nelderMead;
+
+    QUnit.test('getCentroid', assert => {
+        const { getCentroid } = NelderMeadModule;
+        assert.deepEqual(
+            getCentroid([
+                [184.16021264966827, 99.75],
+                [184.16021264966827, 95],
+                [193.3682232821517, 95]
+            ]),
+            [184.16021264966827, 97.375],
+            'should calculate the center point between all the coordinates, except the last'
```

Comment:
Use upper case first for unit test descriptions. Not an explicit style guide requirement, but all existing tests use uc first.